### ISSUE TITLE
Inconsistent variable dim naming in statistics

### ISSUE THREAD

**Issue by julietbravo** [microhh/microhh#230]
The radiation background levels have multiple dimension names in the statistics:

    [bart@manjaro cabauw]$ ncdump -h cabauw.default.0000000.nc 
    netcdf cabauw.default.0000000 {
    dimensions:
        ...
        p_rad = 137 ;
        era_levels = 137 ;

First, `era` should be removed, but perhaps it is anyhow better to rename them to `lay`/`lev`, consistent with the case input?

**Comment by Chiil** [comment]
Agree, that is consistent with the radiation solver, or `p_lay` and `p_lev` as we also use in the stand alone radiation solver output. @MennoVeerman?

**Comment by MennoVeerman** [comment]
> p_lay < and > p_lev < are not used as dimension names in the standalone. The standalone reads dimensions > lay < and < lev > and writes dimensions < n_lay > and < n_lev >, which is also inconsistent. As suggested, I would change everything to < lay > and < lev > (and not > p_lay < and > p_lev <) as we store both pressures and heights on levels and layers

**Comment by MennoVeerman** [comment]
`p_lay` and `p_lev` are not used as dimension names in the standalone. The standalone reads dimensions `lay` and `lev` and writes dimensions `n_lay` and `n_lev`, which is also inconsistent. As suggested, I would change everything to `lay` and `lev` (and not `p_lay` and `p_lev`) as we (can) save both pressures and heights on levels and layers

**Comment by julietbravo** [comment]
Okidokie, I'll fix them now, would be nice to not have to change this after 2.0

**Comment by julietbravo** [comment]
Fixed by https://github.com/microhh/microhh/pull/233
